src/plugins/graph/models/graph-model.ts

```diff
@@ -299,6 +299,8 @@ export const GraphModel = TileContentModel
     },
     exportJson(options?: ITileExportOptions) {
       const snapshot = getSnapshot(self);
+      // TODO: Consider updating exportJson for hashing support
```

Comment:
Just want to be sure: you didn't intend to resolve this as part of these changes?

Reply:
I've removed the comment in https://github.com/concord-consortium/collaborative-learning/pull/2628/commits/e79145ded397a82da9bd328f0e7c4d44f918c0b6.  All the properties of the model should be in the hashed content.